packages/ensnode-sdk/src/ens/names.test.ts

```
@@ -30,6 +30,24 @@ describe("names", () => {
     });
   });
 
+  describe("getParentNameFQDN", () => {
+    it("returns FQDN for an empty name", () => {
+      expect(getParentNameFQDN("")).toStrictEqual("");
```

Seems that this case should throw an error, since there's no parent of the ENS root